docs/operate/monitoring/tracing.mdx

```diff
@@ -11,84 +11,71 @@ Restate supports the following tracing features:
 
-An example from the [shopping cart example](https://github.com/restatedev/examples/tree/main/end-to-end-applications/typescript/ecommerce-store):
-![Observability](/img/observability.jpeg)
+<img src="/img/observability.jpeg" width={"60%"}/>
 
-## Setting up Jaeger file exporter
+<Admonition type="info" title={"Filtering traces"}>
+    You can configure a span/event filter in a similar fashion to the [log filter](/operate/monitoring/restate_logging#log-filter) setting the `observability.tracing.filter` configuration entry.
+</Admonition>
 
-If you can't configure a Jaeger agent, you can export traces by writing them to files, using the Jaeger JSON format. To do so, set up the configuration entry `observability.tracing.json_file_export_path` pointing towards the path where trace files should be written.
+<details className={"grey-details"}>
+    <summary>Setting up Jaeger file exporter</summary>
 
-You can import the trace files using the Jaeger UI:
+    If you can't configure a Jaeger agent, you can export traces by writing them to files, using the Jaeger JSON format. To do so, set up the configuration entry `observability.tracing.json_file_export_path` pointing towards the path where trace files should be written.
 
-![Jaeger UI File import](/img/jaeger-import-file.png)
+    You can import the trace files using the Jaeger UI:
 
+    <img src="/img/jaeger-import-file.png" width={"35%"}/>
+</details>
 
 ## Understanding traces
 The traces contain detailed information about the context calls that were done during the invocation (e.g. sleep, one-way calls, interaction with state):
 
-![Understanding traces](/img/jaeger_tour_background_call_handler.png)
+<img src="/img/jaeger_tour_background_call_handler.png" width={"85%"}/>
 
-The initial `ingress_invoke` spans show when the gRPC/Connect HTTP request was received by Restate. The `invoke` span beneath it shows when Restate invoked the service deployment to process the request.
+The initial `ingress_invoke` spans show when the HTTP request was received by Restate. The `invoke` span beneath it shows when Restate invoked the service deployment to process the request.
 
 The tags of the spans contain the metadata of the context calls (e.g. call arguments, invocation id).
 
+<Admonition type="info" title={"One-way call traces are detached from the parent"}>
 When a service invokes another service, the child invocation is linked automatically to the parent invocation, as you can see in the image.
 Note that the spans of one-way calls are shown as separate traces. The parent invocation only shows that the one-way call was scheduled, not its entire tracing span.
-To see this information, search for the trace of the one-way call by filtering on the invocation id tag:
-```
-restate.invocation.id="T4pIkIJIGAsBiiGDV2dxK7PkkKnWyWHE"
-```
+To see this information, search for the trace of the one-way call by filtering on the invocation id tag `restate.invocation.id="T4pIkIJIGAsBiiGDV2dxK7PkkKnWyWHE"`.
```

Comment:
I think the invocation id is no longer correct. Nowadays it should have the form `inv_....`, I believe.